static/app/components/search/sources/commandSource.tsx

```diff
@@ -65,6 +65,19 @@ const ACTIONS: Action[] = [
   },
 ];
 
+process.env.NODE_ENV !== 'production' &&
+  ACTIONS.push({
+    title: t('Open in Production'),
+    description: t('Open the current page in sentry.io'),
+    requiresSuperuser: false,
+    action: () => {
+      const url = new URL(window.location.toString());
+      url.host = 'sentry.io';
```

Comment:
I've added to the client config of what links we expect on frontend: https://github.com/getsentry/sentry/blob/104d0fda4c3dd11e5a36bd307d5eed907567d9c5/static/app/types/system.tsx#L119-L127

Rather than hardcoding `sentry.io` throughout the frontend app, we expose `window?.__initialData.links.sentryUrl`, so that we can dynamically update the hostname. 

https://github.com/getsentry/sentry/blob/104d0fda4c3dd11e5a36bd307d5eed907567d9c5/static/app/types/system.tsx#L143-L147

I've also exposed constants to determine if a customer domain is to be used: https://github.com/getsentry/sentry/blob/master/static/app/constants/index.tsx#L13-L14

------

With the above, you could make these changes:

```tsx
const hostname = usingCustomerDomain ? 
  window?.__initialData?.customerDomain?. organizationUrl : 
  window?.__initialData.links.sentryUrl;
```

```suggestion
      url.host = hostname;
```

Reply:
Added! I had to use another URL object to get the hostname though since those links seem to contain the protocol as well. Thanks for the detailed comment